### PR TITLE
Modification in order to have a clang out the box 

### PR DESCRIPTION
**Pull request by Lambourl**

### CODE REVIEW

File: .nxxm/deps [hunk 1]
```diff
@@ -1,10 +1,17 @@
 {
-   "daminetreg/js-bind" : { "@" : "v0.0.2" }
+   "nxxm/js-bind" : { "@" : "master" }
```
Comment (daminetreg) [inline .nxxm/deps:2]: please put a particular version

File: .nxxm/deps [hunk 2]
```diff
@@ -1,10 +1,17 @@
...
   , "platform:clang-cxx17" : [ "Boost::system", "OpenSSL::+SSL", "OpenSSL::+Crypto" ]
   , "platform:vs-15-2017-win64-cxx17" : [ "Boost::system", "OpenSSL::+SSL", "OpenSSL::+Crypto", "target::crypt32" ]
   , "platform:vs-16-2019-win64-cxx17" : [ "Boost::system", "OpenSSL::+SSL", "OpenSSL::+Crypto", "target::crypt32" ]
+  
+  , "platform:linux" : [ "Boost::system", "BoringSSL::+ssl", "BoringSSL::+crypto", "BoringSSL::+decrepit", "target::dl" ]
+  , "platform:linux-cxx17" : [ "Boost::system", "BoringSSL::+ssl", "BoringSSL::+crypto", "BoringSSL::+decrepit", "target::dl" ]
+  , "platform:macos" : [ "Boost::system", "BoringSSL::+ssl", "BoringSSL::+crypto", "BoringSSL::+decrepit" ]
+  , "platform:macos-cxx17" : [ "Boost::system", "BoringSSL::+ssl", "BoringSSL::+crypto", "BoringSSL::+decrepit" ]
+  , "platform:windows" : [ "Boost::system", "BoringSSL::+ssl", "BoringSSL::+crypto", "BoringSSL::+decrepit", "target::crypt32" ]
+  , "platform:windows-cxx17" : [ "Boost::system", "BoringSSL::+ssl", "BoringSSL::+crypto", "BoringSSL::+decrepit", "target::crypt32" ]
```
Comment (daminetreg) [inline .nxxm/deps:15]: Isn't it possible here to just write `"platform" : [ "Boost::system", "BoringSSL::+ssl", "BoringSSL::+crypto", "BoringSSL::+decrepit"]`
`"platform:windows" : [ "target::crypt32"]`
?

It it doesn't then we need to fix it in nxxm (create an issue for it)

**Review by daminetreg — CHANGES_REQUESTED** [review]

File: .nxxm/deps [hunk 3]
```diff
@@ -1,10 +1,17 @@
 {
-   "daminetreg/js-bind" : { "@" : "v0.0.2" }
+   "nxxm/js-bind" : { "@" : "v0.0.0" }
```
Comment (daminetreg) [inline .nxxm/deps:2]: oh!!! v0.0.3

File: .nxxm/deps (hunk 2 above)
Comment (daminetreg) [reply .nxxm/deps:15]: So from ad hoc discussion I remember we have a bug there and it isn't possible.

Comment (daminetreg) [reply .nxxm/deps:15]: Issue created at https://github.com/nxxm/nxxm-src/issues/211

File: .nxxm/deps (hunk 3 above)
Comment (Lambourl) [reply .nxxm/deps:2]: On nxxm/js-bind there was no version so I created the first version of se repo so v0.0.0. So do you want me to upgrade it to v0.0.3 ?

Comment (daminetreg) [reply .nxxm/deps:2]: I'm doing it right now, there were 3 version : 

image

**Review by daminetreg — APPROVED** [review]

**Pull request merged**